scripts/security-cleanup.sql

```
@@ -0,0 +1,28 @@
+  'spam1787295195036-1@example.com',
+  'spam1787295195036-2@example.com',
+  'spam1787295195036-3@example.com'
+);
+
+-- Sweep any other never-verified example.com signups older than a week that
+-- have no sessions and no posts (adjust to taste before running).
+DELETE FROM "user" u
+WHERE u."email" LIKE '%@example.com'
+  AND u."emailVerified" = false
+  AND u."createdAt" < now() - interval '7 days'
+  AND NOT EXISTS (SELECT 1 FROM "session" s WHERE s."userId" = u."id")
```

SQLFluff (4.2.2)

[error] The 'WHERE' keyword should always start a new line.

(LT14)